CHANGELOG.md

```diff
@@ -1,35 +1,24 @@
 # Changes
 
-## 2.9.0 [2023/XX/XX] "xxx Release"
-
-**New Features:**
-
-- XXX
-
-**New Features:**
-
-- XXX
+## 2.9.0 [2023/09/22] "Autumn Rain Release"
 
 **Enhancements:**
 
 - Added an error message to indicate that when you can't load evtx files in Windows due to specifying a directory path with spaces in it, you need to remove the trailing backslash. (#1166) (@hitenkoku, thanks for the suggestion from @joswr1ght)
-- Optimized the number of records to load at a time. (#1175) (@yamatosecurity)
-- Replaced double backslashes in paths in under the progress bar on Windows systems with single forward slashes. (#1172) (@hitenkoku)
+- Optimized the number of records to load at a time for performance. (#1175) (@yamatosecurity)
+- Replaced double backslashes in paths under the progress bar on Windows systems with single forward slashes. (#1172) (@hitenkoku)
+- Made the `Details` field for `count` rules a string in the JSON output for easier parsing. (#1179) (@hitenkoku)
 
 **Bug Fixes:**
 
-- XXX
-
-**Other:**
-
-- XXX
+- In rare cases, JSON fields would not be correctly parsed. (#1145) (@hitenkoku)
```

Comment:
Since this description doesn't seem to solve the problem yet, how about the following description?

```suggestion
- Fixed JSON fields would not be correctly parsed in rare cases. (#1145) (@hitenkoku)
```